frontend/document/credocument.lua

```diff
@@ -306,6 +306,14 @@ function CreDocument:getScreenBoxesFromPositions(pos0, pos1, get_segments)
     return line_boxes
 end
 
+function CreDocument:getNextVisibleWordEnd(pos0, pos1)
+    return self._document:nextVisibleWordEnd(pos0, pos1)
```

Comment:
The cre.cpp functions you added and are calling here only eat one parameter (after the credocument that is self._document, implicitely added by the `:` and eaten by `CreDocument *doc = (CreDocument*) luaL_checkudata(L, 1, "credocument");`):
 `const char* xp = luaL_checkstring(L, 2)`
so you shouldn't have `pos0` and `pos1` here, only a single one, better named `xp`.